crowdsourcing/simple_static_task/server_files/craftassist_task.html

```diff
@@ -0,0 +1,122 @@
+<!-- Bootstrap v3.0.3 -->
+<link
+  rel="stylesheet"
+  href="https://stackpath.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
+/>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+<!-- Modal --><!-- MODAL -->
+
+<div
+  aria-hidden="true"
+  aria-labelledby="exampleModalLabel"
+  class="modal fade in"
+  id="exampleModal"
+  role="dialog"
+  tabindex="-1"
+>
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button
+          aria-label="Close"
+          class="close"
+          data-dismiss="modal"
+          type="button"
+        >
+          <span aria-hidden="true"> &times; </span>
+        </button>
+      </div>
+
+      <div class="modal-body" id="modal">&nbsp;</div>
+
+      <div class="modal-footer">
+        <button class="btn btn-secondary" data-dismiss="modal" type="button">
+          Close
+        </button>
+      </div>
+    </div>
+  </div>
+</div>
+
+<!-- Content -->
+
+<section
+  class="container"
+  id="Other"
+  style="
+    margin-bottom: 15px;
+    padding: 10px;
+    font-family: Verdana, Geneva, sans-serif;
+    font-size: 0.9em;
+  "
+>
+  <div class="row col-xs-12 col-md-12">
+    <!-- Instructions -->
+    <div class="panel panel-primary">
+      <div class="panel-heading">
+        <strong> Instructions </strong>
+      </div>
+
+      <div class="panel-body" id="instructions">
+        <p>
+          Here is a sample craftassist task.
+        </p>
+      </div>
+    </div>
+    <!-- End Instructions -->
+    <!-- Content Body -->
+
+    <section>
+      <fieldset>
+        <div class="input-group">
+          <label>Rate your experience with playing with the bot</label>
+          <select class="form-control" name="rating">
+            <option selected="selected" value="">(select one)</option>
+            <option value="good">Good</option>
+            <option value="passable">Passable</option>
+            <option value="bad">Bad</option>
+          </select>
+        </div>
+      </fieldset>
+    </section>
+    <!-- End Content Body -->
+  </div>
+</section>
+
+<section
+  class="container"
+  id="Other"
+  style="
+    margin-bottom: 15px;
+    padding: 10px;
+    font-family: Verdana, Geneva, sans-serif;
+    font-size: 0.9em;
+    height: 1000px;
+  "
+>
+<iframe style="width:100%; height:100%;" src="https://craftassist.io/"></iframe>
```

Comment:
pass work&assignment id to dashboard here.

Reply:
Will need to come in a future PR because I need to apply some recent commits from Mephisto.